the-new-octal/src/app/page/edit-bill/edit-bill.component.scss

```diff
@@ -0,0 +1,11 @@
+.validation {
+  font-style: italic;
+  font-size: 90%;
+
+  &-danger {
+    color: red;
+  }
+  &-success {
+    color: green;
+  }
+}
```

Comment:
A globális css-be emeld ki, mert ismétlés
```suggestion
```

Reply:
kész